src/flux_differences_HLLE.F90

```diff
@@ -199,6 +224,6 @@ subroutine flux_differences_hlle
   
   call cpu_time(t2)
 
-  t_fdhlle = t_fdhlle + (t2-t1)
+!  t_fdhlle = t_fdhlle + (t2-t1)
```

Comment:
is there a reason to comment this out?

Reply:
I changed it back. I did it because when I was debugging some flags would cause the code to stop here since t_fdhlle is never initialized, because start_timers is never called. So commenting this out was easier than calling start_timers I guess